packages/orchestration/src/exos/chain-account-kit.js

```diff
@@ -163,10 +163,12 @@ export const prepareChainAccountKit = (zone, { watch, asVow }) =>
           this.state.connection = connection;
           this.state.remoteAddress = remoteAddr;
           this.state.localAddress = localAddr;
+          const address = findAddressField(remoteAddr);
+          if (!address) {
+            console.error('🚨 failed to parse chain address', remoteAddr);
```

Comment:
the red light is for "page somebody right away; an invariant was violated" per https://github.com/Agoric/agoric-sdk/wiki/Logging

But this can be triggered if the remote end mis-behaves. Seems like this should be a warning.

> ⚠️ indicates something that is not expected but was recovered from.